starfish/intensity_table.py

```diff
@@ -100,7 +103,14 @@ def empty_intensity_table(
         hyb_index = np.arange(n_hyb)
         data = np.zeros((spot_attributes.shape[0], n_ch, n_hyb))
         dims = (IntensityTable.Constants.FEATURES.value, Indices.CH.value, Indices.HYB.value)
-        return cls(data=data, coords=(spot_attributes, channel_index, hyb_index), dims=dims)
+        attrs = {'image_shape': image_shape}
+
+        intensity_table = cls(
+            data=data, coords=(spot_attributes, channel_index, hyb_index), dims=dims,
```

Comment:
hyb_index -> round_index

Reply:
You'll see lots of those -- the PR that makes these changes comes _after_ this one.